src/neo/SmartContract/TriggerType.cs

```diff
@@ -5,24 +5,25 @@ namespace Neo.SmartContract
     [Flags]
     public enum TriggerType : byte
     {
-        System = 0x01,
+        SystemPersist = 1 << 1,
+        SystemPostPersist = 1 << 2,
         /// <summary>
         /// The verification trigger indicates that the contract is being invoked as a verification function.
         /// The verification function can accept multiple parameters, and should return a boolean value that indicates the validity of the transaction or block.
         /// The entry point of the contract will be invoked if the contract is triggered by Verification: 
         ///     main(...);
         /// The entry point of the contract must be able to handle this type of invocation.
         /// </summary>
-        Verification = 0x20,
+        Verification = 1 << 3,
         /// <summary>
         /// The application trigger indicates that the contract is being invoked as an application function.
         /// The application function can accept multiple parameters, change the states of the blockchain, and return any type of value.
         /// The contract can have any form of entry point, but we recommend that all contracts should have the following entry point:
         ///     public byte[] main(string operation, params object[] args)
         /// The functions can be invoked by creating an InvocationTransaction.
         /// </summary>
-        Application = 0x40,
+        Application = 1 << 4,
 
-        All = System | Verification | Application
+        All = SystemPersist | SystemPostPersist | Verification | Application
```

Comment:
@erikzhang I think that we can remove flags from `TriggerType`, what do you think?

Reply:
What do you mean?

Reply:
We never use `TriggerType` as a flag

Reply:
We have used flags to determine `AllowedTriggers`. Maybe it will be needed in the future?